src/FOSRestExtraBundle/EventListener/ParamFetcherListener.php

```diff
@@ -103,10 +79,11 @@ public function onKernelController(ControllerEvent $event): void
                 }
 
                 if (!empty($invalidParams) && $this->isExtraParametersCheckRequired($event)) {
+                    $route = $request->attributes->get('_route');
                     $msg = sprintf(
                         "Invalid parameters '%s' for route '%s'",
                         implode(', ', $invalidParams),
-                        $request->attributes->get('_route')
+                        is_string($route) ? $route : 'unknown'
```

Comment:
**suggestion (non-blocking):** is using the second parameter of the `ParameterBag` do the job ? 🤷 

```suggestion
$request->attributes->get('_route', 'unknown')
```

Reply:
If I keep using it, PHPStan returned 
 ```
  Parameter #3 ...$values of function sprintf expects bool|float|int|string|null, mixed given.  
```